services/github/src/utils.js

```diff
@@ -0,0 +1,33 @@
+const request = require('superagent');
+const util = require('util');
+
+const setTimeoutPromise = util.promisify(setTimeout);
+
+const throttleRequest = async ({url, method, delay = 10, response = {status: 0}, attempt = 1}) => {
+  if (attempt > 5) {
+    return response;
+  }
+
+  let res;
+  try {
+    res = await request(method, url);
+  } catch (e) {
+    if (e.status >= 400 && e.status < 500) {
+      return e;
+    }
+    if (e.status >= 500) {
+      const newDelay = 2 ** attempt * 100;
+      /* eslint-disable comma-dangle */
```

Comment:
Let's not disable these except for exceptional cases. If we don't like the settings, we can change them.

Reply:
This is an exceptional case tho. There's no need in a comma after the last function argument. Or can you explain a bit more what you meant?

Reply:
Nevermind, I reformatted the code another way